src/util/hass-util.html

```diff
@@ -23,7 +24,7 @@
 ];
 
 window.hassUtil.DOMAINS_WITH_MORE_INFO = [
-  'alarm_control_panel', 'automation', 'camera', 'climate', 'configurator',
+  'alarm_control_panel', 'automation', 'binary_sensor', 'camera', 'climate', 'configurator',
```

Comment:
Since `binary_sensor` doesn't have a more info card, I think this change made binary sensors loose their state attributes showing in the frontend. I don't think that was intentional.